api/config.py

```diff
@@ -33,6 +49,18 @@ def map_config(config, options):
     }
 
 
+def processor_info(config):
+    has_been_configured = bool(config["App"]["HasBeenConfigured"])
+    device = config["Detector"]["Device"]
+    if config["Detector"]["Name"] == "openvino":
+        device += "-openvino"
+    return {
+        "version": "0.2.0",
```

Comment:
Can we move the version number to an independent file?
It should be in a place easy to find and edit each time we deploy a new one.